### PR TITLE
multiple image support

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/components/organisms/OrganismImageGallery.vue [hunk 1]
```diff
@@ -26,11 +26,15 @@ export default {
...
             imageSize: '',
+            spotImage: [
+                'https://parkspot.blob.core.windows.net/assets/default.png',
+            ],
         };
     },
     computed: {
         ...mapState('sdp', {
-            spotImage: (state) => state.image,
+            image: (state) => state.image,
```
Comment (arrogant-ant) [inline src/components/organisms/OrganismImageGallery.vue:36]: please rename this to images
to highlight that its an array

File: src/store/sdp/index.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ const state = {
     isAvailable: false,
     loading: false,
     title: '',
+    thumbnail: [],
```
Comment (arrogant-ant) [inline src/store/sdp/index.js:10]: thumbnail should be a single photo
right?

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/components/organisms/OrganismImageGallery.vue (hunk 1 above)
Comment (alias01) [reply src/components/organisms/OrganismImageGallery.vue]: done

File: src/store/sdp/index.js (hunk 2 above)
Comment (alias01) [reply src/store/sdp/index.js:10]: since the gallery component only takes array

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**